contrib/backporting/cherry-pick

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+set -e
+if [ $# -ne 1 ]; then
+  echo "Usage: $0 <commit-id>"
+  exit 1
+fi
+CID=$1
+REM=`git remote -v | grep "github.com.cilium/cilium" | head -n1 | cut -f1`
+BRANCHES=`git branch -q -r --contains $CID $REM/master 2> /dev/null`
+if ! echo ${BRANCHES} | grep -q ".*$REM/master.*"; then
+  echo "Commit $CID not in $REM/master!"
+  exit 1
+fi
```

Comment:
+1 for this check

Reply:
If the wrong one was cherry-picked then you can always reset the HEAD back, and in case of conflict git will already tell you what to do and how to proceed (fix conflict, git add ... , git am --continue).